code/src/index.css

```diff
@@ -5,9 +5,107 @@ body {
     sans-serif;
   -webkit-font-smoothing: antialiased;
   -moz-osx-font-smoothing: grayscale;
+  background-color: black;
+}
+
+html {
+  font-size: 62.5%;
+  color: #fff;
 }
 
 code {
   font-family: source-code-pro, Menlo, Monaco, Consolas, "Courier New",
     monospace;
 }
+
+h2 {
+  font-size: 4.8rem;
+}
+
+
+.container {
+  gap: 10rem;
+  margin: 4rem 2rem;
+}
+
+.displayAlbum {
+  display: grid;
+  grid-template-columns: repeat(auto-fit, minmax(35rem, 1fr));
```

Comment:
We noticed that in the desktop view it is showing only three images when in the instructions they are requiring 4 images for desktop. The mobile and tablet view works but we think you could add a media query for the grid-template-columns in desktop view (starting for example from 1024px). Right now your site breaks into 3 images in 1200px and into 4 images at 1500px which is very big. :)